src/test/java/seedu/address/commons/core/ReminderTest.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.commons.core;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Date;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Contains unit test for Reminder.
+ */
+
+class ReminderTest {
+
+    @Test
+    public void getReminderMessage() {
+        Reminder dummyReminder = new Reminder("Pay Bill", new Date(2019, 10, 10));
+        assertEquals("Pay Bill", dummyReminder.getReminderMessage());
+    }
+
+    @Test
+    public void getDeadline() {
```

Comment:
change test to `isValidDeadline()` or merge it with `isValidReminder()`

Reply:
why should it be separated? I think it should be separated as they test different things.